stance/skeleton/skeleton.py

```diff
@@ -1,43 +1,71 @@
 import numpy as np
-from collections import namedtuple
-
-COCOPts = namedtuple('COCOPts', ('nose',
-                                 'neck',
-                                 'right_shoulder',
-                                 'right_elbow',
-                                 'right_wrist',
-                                 'left_shoulder',
-                                 'left_elbow',
-                                 'left_wrist',
-                                 'right_hip',
-                                 'right_knee',
-                                 'right_ankle',
-                                 'left_hip',
-                                 'left_knee',
-                                 'left_ankle'))
-
-
-class Skeleton:
-    def __init__(self, coco_points):
+from typing import Dict, Tuple, Sequence
+
+
+COCOPts = Sequence[Tuple[float, float]]
+
+
+class SKVector(object):
+    def __init__(self, head: Tuple[float, float], tail: Tuple[float, float]) -> None:
+        self.head: np.ndarray = np.array(head)
+        self.tail: np.ndarray = np.array(tail)
+        self.vec = self.head - self.tail
+
+
+    def cos_similarity(self, other: 'SKVector') -> float:
+        """Computes the cosine similarity between this vector and another
```

Comment:
There should be a newline after """

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Reply:
🤮 ok but its like this everywhere else in the code, can we keep it similar for now and change the formatting of the whole system later :) pleeeeease